toolchain/check/import_cpp.cpp

```diff
@@ -187,6 +188,43 @@ static auto ClangLookup(Context& context, SemIR::LocId loc_id,
   return lookup;
 }
 
+// Returns the return type of the given function declaration.
+// Currently only void and 32-bit int are supported.
+// TODO: Support more return types.
+static auto GetReturnType(Context& context,
+                          const clang::FunctionDecl* clang_decl)
+    -> SemIR::InstId {
+  auto ret_type = clang_decl->getReturnType().getCanonicalType();
+  if (ret_type->isVoidType()) {
+    return SemIR::InstId::None;
+  }
+  const int supported_int_width = 32;
```

Comment:
Had you considered marking this `constexpr` instead of just `const`? This has some different effects, including different naming checks for clang-tidy since it's a stronger guarantee.

```suggestion
  constexpr int SupportedIntWidth = 32;
```

Reply:
Thanks. This line is removed in the new version.